### ISSUE TITLE
"latest" showing at top left corner of page

### ISSUE THREAD

**Issue by RogerKIWI** [sayontan/suffusion#68]
Suffusion 4.6.1 downloaded, although when installed it identified as 4.6.0 until I changed the 3 instances of 4.6.0 to 4.6.1

I don't know if I've done something wrong, or there's something I'm missing...

<img width="554" alt="image" src="https://github.com/sayontan/suffusion/assets/2053916/0aabfb3c-c43c-4443-bec8-e74fe2f497a4">

I'm seeing the word latest at the very top left corner of any page on my new site I'm developing for my genealogy (to try out a different plugin for displaying the genealogy data).

https://testing.lisaaandroger.com/

running WordPress 6.2.3, and it also shows on another site running WordPress 6.4.2 at 

https://new.lisaandroger.com/

It shows in the source code of a page here

<img width="837" alt="image" src="https://github.com/sayontan/suffusion/assets/2053916/ab7fcc46-4273-4744-9df8-00c414bda9cc">

but I can not figure out where it's coming from - it's from Suffusion somewhere as it goes away if I change the theme, but I can't figure out where it's coming from.

Is anyone else seeing this, and can point me to where it's coming from so I can remove it?

Thanks

Roger

**Comment by marcissimus** [comment]
Hi Roger,

sorry, I cut the 4.6.1 zip file before updating the version. It's updated now. 

I just released 4.6.2 which has the correct version number (and fixes a few log warnings). Give that a try.

I don't see the issue with "latest" on my suffusion sites. Are you using a child theme? 

Best,
Marc

**Comment by marcissimus** [comment]
I found the cause of "latest" 😀  I will include it in 4.6.2.

**Comment by marcissimus** [comment]
Latest 4.6.2 should fix this issue. Please test 🙏

**Comment by mbrsolution** [comment]
Thank you @marcissimus for the latest fix. I was going nuts trying to find the word "Latest" in the code. I can confirm version 4.6.2 removes the word "Latest".

Well done :)

**Comment by RogerKIWI** [comment]
> On Dec 16, 2023, at 4:50 PM, Marc Bernard ***@***.***> wrote:
> 
> 
> Latest 4.6.2 should fix this issue. Please test 🙏
> 
> 
Thank You!!

I’ll check it out later this evening - dealing with car problems right now.

Roger

**Comment by RogerKIWI** [comment]
> On Dec 16, 2023, at 5:17 PM, mbrsolution ***@***.***> wrote:
> 
> Thank you @marcissimus <https://github.com/marcissimus> for the latest fix. I was going nuts trying to find the word "Latest" in the code. I can confirm version 4.6.2 removes the word "Latest".
> 


I almost went nuts LOL so decided to ask for help.

Thanks @marcissimus I’ll check it out later

Roger

**Comment by mbtools** [comment]
I'm on here nearly every day, so don't hesitate to open an issue.

**Comment by mbrsolution** [comment]
Thank you for helping so much, I really appreciate this. I was trying to find the issue before I reported it. I want to help in any way I can. Plus diving into the code also helps me understand PHP as well, which I want to become more familiar with :)

Kind regards,
Manuel

**Comment by mbtools** [comment]
PHP is still a hobby for me but I learned a lot from many plug-ins and themes over the years. The key for understanding how WP works is learning about hooks and templates. 

The issue here was related to a hook, some pho that the theme "told" WP to call. This overwrites the usual WP behaviour (via function.php in the theme). I initially didn't understand exactly how this particular hook worked and returned the wrong data. Well now I learned something, too, and can use the hook correctly. 👍
